opentaxii/defaults.yml

```diff
@@ -7,6 +7,7 @@ save_raw_inbox_messages: yes
 xml_parser_supports_huge_tree: yes
 count_blocks_in_poll_responses: no
 return_server_error_details: no
+unauthorized_status: UNAUTHORIZED
```

Comment:
could we add a comment explaining what options exist for this setting?

Reply:
I've added validation into Config reader and updated docs :+1: